### DISCUSSION TITLE
php.ini date.timezone issue

### DISCUSSION THREAD

**Discussion by akagumo** [shivammathur/homebrew-php#961] [Q&A / Help]
php 8.x is being strict about date.timezone.

the php.ini that is included with the package has the wrong date.timezone.
this date.timezone = “US/Pacific” is in the current php.ini
notice the depreciated US/Pacific and also the curly double quotes.
php's own site has a space, and php will error if there are any spaces so use underscore.
some were suggesting US, but US errors for me. https://www.php.net/manual/en/timezones.america.php
below is what works
date.timezone = "America/Los_Angeles"

**Comment by shivammathur** [comment]
The package does not set up a timezone in php.ini actually. It uses the default php.ini from the source.
https://github.com/Homebrew/homebrew-core/blob/master/Formula/php.rb#L217

You can do that yourself.
